### PR TITLE
Feature/#2795 ubs admin cabinet order client info

### PR DESCRIPTION
**Pull request by YuriShevchuk**
add order-client-info component in ubs-admin-cabinet page
Screenshot_1

### CODE REVIEW

**Review by rostikdoroshenko — APPROVED** [review]

File: src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts [hunk 1]
```diff
@@ -210,6 +206,6 @@ export class UBSPersonalInformationComponent extends FormBaseComponent implement
     this.orderService
       .processOrder(this.order)
       .pipe(takeUntil(this.destroy))
-      .subscribe((val) => (this.shareFormService.orderUrl = this.getStringBetween(val, 'href="', '"')));
+      .subscribe((val) => (this.shareFormService.orderUrl = val.toString()));
```
Comment (JimHolman1199) [inline src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts:209]: so you will create new subsription every time the user click submit button ?

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  background: var(--ubs-primary-white);
+  width: 177px;
+  margin-bottom: 14px;
+  border: 1px solid var(--ubs-primary-green);
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+.violation__item {
+  margin-right: 38px;
+}
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.scss:40]: let's add an empty space after each scss rule

Comment (JimHolman1199) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.scss:40]: also, provide PR description

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+      <div class="form__block col-md-4 col-sm-6 offset-md-1">
+        <div class="form-group">
+          <p class="form__subtitle">Отримувач</p>
+          <label class="form__label" for="name">Ім'я</label>
+          <input class="form-control" type="text" id="name" placeholder="Placeholder" />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="name">Телефон</label>
+          <input class="form-control" type="text" id="tel" placeholder="Placeholder" />
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html:33]: Change 'for' attribute from 'name' to 'tel'

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="name">Email</label>
+          <input class="form-control" type="text" id="email" placeholder="Placeholder" />
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html:37]: Change 'for' attribute from 'name' to 'email'

**Review by oliachipko — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 5]
```diff
@@ -62,6 +63,10 @@ export const routes: Routes = [
     path: 'ubs-admin-cabinet',
     component: UbsAdminCabinetComponent
   },
+  {
+    path: 'ubs-admin-order-client-info',
+    component: UbsAdminOrderClientInfoComponent
+  },
```
Comment (JimHolman1199) [inline src/app/app-routing.module.ts:69]: pull last dev branch and update this route

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html (hunk 4 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html]: ✅

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html (hunk 3 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html]: ✅

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.scss (hunk 2 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.scss:46]: ✅

**Review by Edw1x — APPROVED** [review]

**Review by andreikazmirovich — CHANGES_REQUESTED** [review]

**Pull request closed without merging**